sirepo/job_supervisor.py

```diff
@@ -710,11 +708,19 @@ def _set_error(op, compute_job_serial):
                 raise
             # There was a timeout getting the run started. Set the
             # error and let the user know. The timeout has destroyed
-            # the op so don't need to in _set_error
-            _set_error(None, c)
+            # the op so don't need to
```

Comment:
unfinished sentence. "need to destroy here."

Reply:
https://github.com/radiasoft/sirepo/pull/2523/commits/2c505a1eca579dafbef6921f946a8bc6a642cdc8